docs/overview.md

```diff
@@ -1,19 +1,19 @@
 # Kubernetes User Documentation
 
-The Kubernetes API currently manages 3 main resources:
-* [pods](pods.md)
-* [replication controllers](replication-controller.md)
-* [services](services.md)
+Kubernetes is an open source orchestration system for containers (currently Docker containers). It schedules containers onto nodes in a compute cluster and actively manages these containers to ensure that the state of the cluster continually matches the user's intentions.
```

Comment:
* Fixed hyphenation.
* Changed to "application containers".
* Replaced sentence that talked about container orchestration, with sentence you mentioned from top-level README.
* Incorporated all the other stuff you suggested except the stuff about how you think about your application. I was unable to write anything that didn't sound like total fluff (raises the level of abstraction blah blah blah). I think this is a case of "less is more."